code/script.js

```diff
@@ -1,47 +1,160 @@
-// All the DOM selectors stored as short variables
+// Declare variables
+const startButton = document.getElementById('start-button');
+const startContent = document.getElementById('start-content');
+const chatBox = document.getElementById('chat-box');
 const chat = document.getElementById('chat')
+const form = document.getElementById('name-form')
+const nameInput = document.getElementById('name-input')
+const inputWrapper = document.getElementById('input-wrapper');
 
-// Global variables, if you need any, declared here
+let buttonText = "";
+let questionCounter = 0;
 
-// Functions declared here
+// Start chat by moving past start page
+startButton.addEventListener('click', () => {
+  startContent .style.display = "none";
+  chatBox.style.display = "flex";
+  questionGenerator();
+});
 
-// This function will add a chat bubble in the correct place based on who the sender is
+// Function to move through questions
+const questionGenerator = () => {
+  questionCounter++;
+  if (questionCounter === 1) {
+    setTimeout(question1, 1000);
+  } else if (questionCounter === 2) {
+    setTimeout(question2, 1000, nameInput.value);
+  } else if (questionCounter === 3) {
+    setTimeout(question3, 1000, buttonText);
+  } else if (questionCounter === 4) {
+    setTimeout(question4, 1000);
+  } else if (questionCounter === 5) {
+    setTimeout(question5, 1000);
+  } else if (questionCounter === 6) {
+    setTimeout(endMessage, 1000, nameInput.value);
+  }
+}
+
+// Question 1
+const question1 = () => {
+  showMessage(`Hello there, what's your name?`, 'bot')
+
+  document.getElementById('name-form').onsubmit = event => {
+    event.preventDefault();
```

Comment:
Do you think it is better to move the onsubmit listener outside of the question1 function? I think it could be moved to the bottom of your JS file where you have the other event listeners.